kaspresso/src/main/kotlin/com/kaspersky/kaspresso/systemsafety/SystemDialogSafetyPattern.kt

```diff
@@ -0,0 +1,9 @@
+package com.kaspersky.kaspresso.systemsafety
+
+import java.util.regex.Pattern
+
+enum class SystemDialogSafetyPattern(val pattern: Pattern) {
```

Comment:
Old \\S*google.android\\S* can't cover all cases that we need, so i'm adding enum with multiple patterns, which anyone can extend.